recipes/mysql-concpp/all/test_package/CMakeLists.txt

```
@@ -0,0 +1,8 @@
+cmake_minimum_required(VERSION 3.15)
+project(PackageTest CXX)
+
+find_package(mysql-concpp REQUIRED CONFIG)
+
+add_executable(${PROJECT_NAME} src/example.cpp)
+target_link_libraries(${PROJECT_NAME} PRIVATE mysql::concpp)
+
```

```suggestion
target_compile_features(${PROJECT_NAME} PRIVATE cxx_std_17)
```
Might help with the Conan v1 issues.